pkg/build/graph/analysis/bc.go

```diff
@@ -98,3 +102,79 @@ func FindCircularBuilds(g osgraph.Graph) []osgraph.Marker {
 
 	return markers
 }
+
+func FindPendingTags(g osgraph.Graph) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+ist:
+	for _, uncastIstNode := range g.NodesByKind(imagegraph.ImageStreamTagNodeKind) {
+		istNode := uncastIstNode.(*imagegraph.ImageStreamTagNode)
+		if bcNode, points := buildPointsToTag(g, uncastIstNode); points && !istNode.Found() {
+
+			buildNode := latestBuild(g, bcNode)
+
+			if buildNode == nil {
+				// A build config points to the non existent tag but no current build exists.
+				markers = append(markers, osgraph.Marker{
+					Node:         bcNode,
+					RelatedNodes: []graph.Node{uncastIstNode},
+
+					Severity:   osgraph.WarningSeverity,
+					Key:        TagNotAvailableWarning,
+					Message:    fmt.Sprintf("%s needs to be imported or created by a build.", istNode.ResourceString()),
+					Suggestion: osgraph.Suggestion(fmt.Sprintf("oc start-build %s", bcNode.(*buildgraph.BuildConfigNode).ResourceString())),
+				})
+				continue ist
+			}
+
+			switch buildNode.(*buildgraph.BuildNode).Build.Status.Phase {
+			case buildapi.BuildPhaseCancelled:
+				// TODO: Add a warning here.
+			case buildapi.BuildPhaseError:
+				// TODO: Add a warning here.
+			case buildapi.BuildPhaseComplete:
+				// We should never hit this. The output of our build is missing but the build is complete.
+				// Most probably the user messed up.
```

Comment:
When the build is complete we should have the tag populated so we should never hit this.  Or even if we do, it's probably a race and we should do nothing I think.